packages/ethereum-storage/package.json

```diff
@@ -43,49 +43,50 @@
     "@requestnetwork/smart-contracts": "0.12.0",
     "@requestnetwork/types": "0.20.0",
     "@requestnetwork/utils": "0.19.0",
-    "bluebird": "3.5.3",
-    "bn.js": "4.11.8",
-    "form-data": "2.3.3",
-    "ipfs-unixfs": "0.3.0",
-    "keyv": "3.1.0",
+    "bluebird": "3.7.2",
+    "bn.js": "5.1.3",
+    "form-data": "3.0.0",
+    "ipfs-unixfs": "2.0.3",
```

Comment:
Did you check if the IPFS files created are still considered valid by the node? I'm afraid it can have an impact on the size calculation, since it's two major versions.